doc/manual/src/projects.md

```diff
@@ -458,6 +458,32 @@ Hydra can send email notifications when the status of a build changes.
 This provides immediate feedback to maintainers or committers when a
 change causes build failures.
 
+The feature can be turned on by adding the following line to `hydra.conf`
+
+    email_notification = 1
+
+Hydra can be forced to send an email for each build using the environment
+variable `HYDRA_FORCE_SEND_MAIL`:
+
+    services.hydra-dev.extraEnv.HYDRA_FORCE_SEND_MAIL = "1";
```

Comment:
Why would someone use the environment variable instead of the configuration value?

Reply:
I'm not aware of any equivalent configuration value. Please note that this isn't a "replacement" for `email_notification = 1` in `hydra.conf` It essentially forces Hydra to send emails even for e.g. succeeded builds (or repeatedly failing builds etc). See also `src/lib/Hydra/Plugin/EmailNotification.pm` at line ~60.

Reply:
But as you seem to misunderstand it, I guess I should rephrase that :)